src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,81 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static String SYSTEM_LINE_SEPARATOR = System.getProperty("line.separator");
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFileToString(fromFileName);
+        content = process(content);
+        writeStringToFile(toFileName, content);
+    }
+
+    private String readFileToString(String fileName) {
+        BufferedReader bufferedReader = null;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(fileName));
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+        String content = "";
+        try {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                content += value + " ";
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Same and add a message to your exception. It will be really wondering to get a just RE